### PR TITLE
Restructure address space and addresses documentation

### PR DESCRIPTION
**Pull request by lulf**
* Group address space, address and user docs into a 'tenant guide'
* Describe AddressSpace and Address resources in more detail
* Group installation, configuration, upgrading and monitoring into a 'service admin guide'
* Replace old getting started guides with a new one that reuse parts of the other guides

Example rendered docs: http://people.redhat.com/~ulilleen/docsbuild1/

### CODE REVIEW

**Comment by lulf** [comment]
@jenmalloy I've rebased this on master with your user doc changes, and used italics for the namespace where appropriate.

File: documentation/common/attributes.adoc [hunk 1]
```diff
@@ -53,7 +53,7 @@
 // Other links
 
 // Section enablers
-:Kubernetes:
+:Kubernetes: Yes
```
Comment (jenmalloy) [inline documentation/common/attributes.adoc]: I think just adding the element here enables it? @ncbaratta can you confirm please?

File: documentation/assemblies/assembly-managing-address-spaces.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+:parent-context: {context}
+
+[id='managing-address-spaces-{context}']
+= Managing address spaces
+
+{ProductName} is configured to support manipulating address spaces using the
+ifdef::Kubernetes[Kubernetes and]
+OpenShift command-line tools.
+
```
[the remark this answers is not in the document]
Comment (lulf) [reply documentation/assemblies/assembly-managing-address-spaces.adoc]: Ah, its really just examples. How about changing the titles to "Example minimal configuration" etc.?

[the remark this answers is not in the document]
Comment (jenmalloy) [reply documentation/assemblies/assembly-managing-address-spaces.adoc]: Oh ok! Yes, that works.

File: documentation/common/attributes.adoc (hunk 1 above)
Comment (lulf) [reply documentation/common/attributes.adoc]: Yeah, that was my bad just testing something!

**Pull request merged**